matlab/filter_design.m

```diff
@@ -0,0 +1,63 @@
+clear all
```

Comment:
There is nothing about this in the Readme, could you add a brief doc on how to set the filter params using this script?

[optional]
Do you think it'd be possible to use scilab for this?